geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DiskStoreCommandsDUnitTest.java

```diff
@@ -319,4 +329,79 @@ public void testCompactDiskStore() throws Exception {
     assertThat(cmdResult.getMapFromSection("server-2").keySet()).contains("UUID", "Host",
         "Directory");
   }
+
+  @Test
+  public void testDescribeDiskStoreCommand() throws Exception {
+    MemberVM server1 = rule.startServerVM(0, x -> x.withJMXManager().withProperty("groups", GROUP));
+
+    gfsh.connectAndVerify(server1.getJmxPort(), GfshCommandRule.PortType.jmxManager);
+
+    createDiskStoreAndRegion(server1, 1);
+
+    server1.invoke(() -> {
+      Cache cache = ClusterStartupRule.getCache();
+      Region r = cache.getRegion(REGION_1);
+      r.put("A", "B");
+    });
+
+    CommandResult result = gfsh.executeCommand(
+        String.format("describe disk-store --member=%s --name=%s", server1.getName(), DISKSTORE));
+    assertThat(result.getStatus()).isEqualTo(Result.Status.OK);
+
+    Map<String, String> data =
+        result.getMapFromSection(DescribeDiskStoreCommand.DISK_STORE_SECTION);
+    assertThat(data.keySet()).contains("Disk Store Name", "Member ID", "Member Name",
+        "Allow Force Compaction", "Auto Compaction", "Compaction Threshold", "Max Oplog Size",
+        "Queue Size", "Time Interval", "Write Buffer Size", "Disk Usage Warning Percentage",
+        "Disk Usage Critical Percentage", "PDX Serialization Meta-Data Stored");
+
+    Map<String, List<String>> directories =
+        result.getMapFromTableContent(DescribeDiskStoreCommand.DISK_DIR_SECTION);
+    assertThat(directories.get("Disk Directory").size()).isEqualTo(1);
+
+    Map<String, List<String>> regions =
+        result.getMapFromTableContent(DescribeDiskStoreCommand.REGION_SECTION);
+    assertThat(regions.get("Region Path").size()).isEqualTo(1);
+  }
+
+  @Test
+  public void testUpgradeOfflineDiskStoreCommandFailsAsExpected() throws Exception {
+    MemberVM server1 = rule.startServerVM(0, x -> x.withJMXManager().withProperty("groups", GROUP));
+
+    gfsh.connectAndVerify(server1.getJmxPort(), GfshCommandRule.PortType.jmxManager);
+
+    createDiskStoreAndRegion(server1, 1);
+
+    server1.invoke(() -> {
+      Cache cache = ClusterStartupRule.getCache();
+      Region r = cache.getRegion(REGION_1);
+      r.put("A", "B");
+    });
+
+    // Should not be able to do this on a running system
+    String diskDirs = new File(server1.getWorkingDir(), DISKSTORE).getAbsolutePath();
+    gfsh.executeAndAssertThat(
+        String.format("upgrade offline-disk-store --name=%s --disk-dirs=%s", DISKSTORE, diskDirs))
+        .statusIsError().containsOutput("The disk is currently being used by another process");
+
+    server1.stop(false);
```

Comment:
Note: When you sync this PR with develop make sure the merges are all correct. When I merged develop to do this review, this line didn't merge correctly. I had to fix locally to build and run tests with the changes.

Reply:
I think think your develop might have been a bit behind.

Reply:
Could be. Thought I had updated in the checkout I was in for reviewing this, but maybe not.